server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -687,6 +688,7 @@ public void run() {
 
     MetricsInfo metricsInfo = getContext().getMetricsInfo();
     metricsInfo.addServiceTags(getApplicationName(), clientAddress);
+    metricsInfo.addCommonTags(List.of(Tag.of("queue", this.queueName)));
```

Comment:
Coul d use resource.group as the tag name, seems scan server is using that in 2.1

https://github.com/apache/accumulo/blob/7147177385ef04961250b63cf5a709c41bd94287/server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java#L408

Reply:
It's likely that this should be removed in the merge to main as resource group is added to the set of common tags for all servers in 4.0 already.

Reply:
I'll backport the metric changes so it matches the 4.0 code.